src/components/tools/tool-tile.tsx

```diff
@@ -216,6 +242,7 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
       >
         {this.renderLinkIndicators()}
         {dragTileButton}
+        {isUserResizable && resizeTileButton}
```

Comment:
As mentioned above, if you test for `isUserResizable` above you wouldn't need to test it here.
```suggestion
        {resizeTileButton}
```